### PR TITLE
Add myself as maintainer

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  sha256: {{ hash }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e0d9930f46d41f086a763f40a12d014c46bbc5e19e728c8d66d7a06fef616739
 
 build:
-  preserve_egg_dir: True
```
Comment (beckermr) [inline recipe/meta.yaml:15]: Is this not needed anymore? Unless we are sure, i'd be better to keep it.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -32,10 +29,14 @@ test:
...
     - zodbpickle
     - zodbpickle.tests
+  commands:
+    - pip check
+  requires: 
+    - pip
 
 about:
   home: http://pypi.python.org/pypi/zodbpickle
-  license: PSF 2 and ZPL 2.1
```
Comment (beckermr) [inline recipe/meta.yaml:38]: Why the change in license?

File: recipe/meta.yaml (hunk 1 above)
Comment (jan-janssen) [reply recipe/meta.yaml:15]: I thought it is no longer needed when we install from pip.

File: recipe/meta.yaml (hunk 2 above)
Comment (jan-janssen) [reply recipe/meta.yaml:38]: Previously the bot complained: License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression. Is there a way to have both licenses?

Comment (beckermr) [reply recipe/meta.yaml:38]: Yes. See the SPDX syntax on their site and in out documentation. https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions

File: recipe/meta.yaml (hunk 1 above)
Comment (beckermr) [reply recipe/meta.yaml:15]: OK. IDK so let's go without it!

**Comment by beckermr** [comment]
@conda-forge-admin relint
